providers/base/units/graphics/jobs.pxu

```diff
@@ -289,6 +289,72 @@ command:
  fi
 _summary: Check the OpenGL renderer (AMD GPU and DRI_PRIME=1, nvidia GPU and __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia)
 
+unit: template
+template-resource: graphics_card
+plugin: shell
+category_id: com.canonical.plainbox::graphics
+id: graphics/{index}_glxgears_auto_{product_slug}
+flags: also-after-suspend
+requires: executable.name == 'glxgears'
+command:
+ # shellcheck disable=SC1091
+ prime_offload_tester.py -c glxgears -p {pci_device_name} -d {driver} -t 30
+_summary: Test that glxgears works for {vendor} {product}
+
+unit: template
+template-resource: graphics_card
+plugin: user-interact-verify
+category_id: com.canonical.plainbox::graphics
+id: graphics/{index}_glxgears_valid_{product_slug}
+flags: also-after-suspend
+requires: executable.name == 'glxgears'
+command:
+ # shellcheck disable=SC1091
+ prime_offload_tester.py -c glxgears -p {pci_device_name} -d {driver} -t 30
+_summary: Test that glxgears works for {vendor} {product}
+_description:
+ PURPOSE:
+     This test tests the basic 3D capabilities of your {vendor} {product} video card
+ STEPS:
+     1. Click "Test" to execute an OpenGL demo. Press ESC at any time to close.
+     2. Verify that the animation is not jerky or slow.
+ VERIFICATION:
+     1. Did the 3d animation appear?
+     2. Was the animation free from slowness/jerkiness?
+
+unit: template
+template-resource: graphics_card
+plugin: shell
+category_id: com.canonical.plainbox::graphics
+id: graphics/{index}_glxgears_auto_fullscreen_{product_slug}
+flags: also-after-suspend
+requires: executable.name == 'glxgears'
+command:
+ # shellcheck disable=SC1091
+ prime_offload_tester.py -c "glxgears -fullscreen" -p {pci_device_name} -d {driver} -t 30
+_summary: Test that glxgears works on fullscreen for {vendor} {product}
+
+unit: template
+template-resource: graphics_card
+plugin: user-interact-verify
+category_id: com.canonical.plainbox::graphics
+id: graphics/{index}_glxgears_valid_fullscreen_{product_slug}
+flags: also-after-suspend
+requires: executable.name == 'glxgears'
+command:
+ # shellcheck disable=SC1091
+ prime_offload_tester.py -c "glxgears -fullscreen" -p {pci_device_name} -d {driver} -t 30
+_summary: Test that glxgears works on fullscreen for {vendor} {product}
```

Comment:
This is the same summary as for the `glxgears_auto_fullscreen` job. Is it expected?